osbenchmark/workload/params.py

```diff
@@ -852,6 +852,7 @@ class VectorSearchParamSource(SearchParamSource):
     def __init__(self, workload, params, **kwargs):
         super().__init__(workload, params, **kwargs)
         self.delegate_param_source = VectorSearchPartitionParamSource(workload, params, self.query_params, **kwargs)
+        self.corpora = self.delegate_param_source.corpora
```

Comment:
Why this line?

Reply:
In order to download corpora , workload looks for param source instance's field name 'corpora'. Here VectorSearchParamSource is considered to paramsource, hence, i have to copy corpora from delegate_param_source to actual param source instance.